osu.Game.Modes.Osu/Objects/Drawables/SliderTicksRenderer.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Framework.Caching;
+using osu.Framework.Graphics.Containers;
+using System.Collections.Generic;
+
+namespace osu.Game.Modes.Osu.Objects.Drawables.Pieces
+{
+    public class SliderTicksRenderer : Container<DrawableSliderTick>

```

Comment:
I wouldn't call this "Renderer" since rendering is usually the act of drawing itself. `SliderTicksContainer` sounds better to me.